### DISCUSSION TITLE
🐛 [Firebase Remote Config] Support for Desktop

### DISCUSSION THREAD

**Discussion by sgshivamgarg8** [firebase/flutterfire#8981] [Feature request] [answered]
Why is remote config not supported for desktop yet?

And is it going to be supported in near future ever?
Are there any plans for it?

**Comment by darshankawar** [comment, marked as answer]
The desktop support seems to be in beta per this table : https://firebase.google.com/docs/flutter/setup?platform=ios , so hopefully it'll reach stable in future releases. I suggest you to keep an eye on the table for reference.
Also, moving this issue to discussion.
